sql/postgres/sqlspec.go

```diff
@@ -226,3 +227,15 @@ var TypeRegistry = specutil.NewRegistry(
 		specutil.TypeSpec("sql", &schemaspec.TypeAttr{Name: "def", Required: true, Kind: reflect.String}),
 	),
 )
+
+var (
+	hclState = schemahcl.New(schemahcl.WithTypes(TypeRegistry.Specs()))
```

Comment:
The TypeRegistry can be private imo.

Reply:
you're probably right. let me consider this a bit more